src/styles/blocks/container.scss

```
@@ -0,0 +1,12 @@
+.container {
+  margin: 0 auto;
+  max-width: 1110px;
+
+  @include onTablet {
+    max-width: 689px;
+  }
+
+  @include onTablet {
```

```suggestion
  @include onMobile {
```

промахнулась названием